lib/helium_config.ex

```diff
@@ -38,8 +37,12 @@ defmodule HeliumConfig do
   def save_organization(new_org = %Core.Organization{}) do
     current =
       case Repo.get(DB.Organization, new_org.oui) do
-        nil -> %DB.Organization{oui: new_org.oui}
-        existing_org -> Repo.preload(existing_org, routes: :devaddr_ranges)
+        nil ->
```

Comment:
I'm a little curious about the direction of having `Organization` functions live directly in the `HeliumConfig` module vs their own module/context like `HeliumConfig.Organization`.